src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/psi/internal/core/providers/GradleProjectLabelProvider.java

```diff
@@ -31,11 +33,17 @@
 public class GradleProjectLabelProvider implements IProjectLabelProvider {
 
 	public static final String GRADLE_LABEL = "gradle";
+	private static final PluginManager instance = PluginManager.getInstance();
+	private static final PluginId gradlePluginId = PluginId.findId("com.intellij.gradle");
```

Comment:
These are not constant expressions, I'd thus preferably not declare them as static. I'd rather use an instance var any maybe do lazy loading.

Reply:
Hi @adietish, thanks for the review.
Actually, this type of conditional qualification is not needed at all. The correct solution is to not make these Gradle dependent providers mandatory, only load them when Gradle plug-in is enabled.
See #755 for the correct and simple solution.